freq2mode.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+import argparse
+import math
+import sys
+from ase.data import atomic_masses
+import ase.io
+
+
+def get_atomic_mass(element_symbol):
+    mass_in_amu = atomic_masses[element_symbol]
+    # Convert mass to atomic units (1 amu = 1.66053904e-27 kg)
+    mass_in_au = mass_in_amu * 1.66053904e-27 / 9.10938356e-31
```

Comment:
Can we use the ASE unit tools for this to not have hard coded value transformations?